### PR TITLE
added "Paste and Match Style"

### PR DESCRIPTION
**Pull request by sndrkrly**
### **Issue #2283**

I can't find where is the "right-click" menu, so 
it's not listed there yet, only on the "topbar" (or what is it called).

### CODE REVIEW

File: main/menu.js [hunk 1]
```diff
@@ -212,6 +212,11 @@ function buildAppMenu (options = {}) {
           accelerator: 'CmdOrCtrl+V',
           role: 'paste'
         },
+        {
+          label: l("pasteAndMatchStyle"),
+          accelerator: "", /* change this to some accelerator */
+          role: "pasteAndMatchStyle"
```
Comment (sndrkrly) [inline main/menu.js:218]: Issue #2283

File: localization/languages/en-US.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "cut": "Cut",
     "copy": "Copy", //this is a verb (as in "copy the currently-selected text")
     "paste": "Paste",
+    "pasteAndMatchStyle": "Paste and Match Style",
```
Comment (PalmerAL) [inline localization/languages/en-US.json:26]: Can you also add this line to the other language files, so the translators know to add it?

```
    "pasteAndMatchStyle": null, //missing translation
```

File: main/menu.js [hunk 3]
```diff
@@ -212,6 +212,11 @@ function buildAppMenu (options = {}) {
           accelerator: 'CmdOrCtrl+V',
           role: 'paste'
         },
+        {
+          label: l("pasteAndMatchStyle"),
+          accelerator: "", /* change this to some accelerator */
```
Comment (PalmerAL) [inline main/menu.js:217]: This opt+shift+cmd+v on macOS, so `alt+shift+CmdOrCtrl+v"? Not sure what it is on Windows though.

**Review by PalmerAL — COMMENTED** [review]
Thanks!

The right-click menu is `js/webviewMenu.js`. There's also `js/contextMenu.js`, but that's really only used in the address bar, which doesn't accept styled text anyway, so I don't think it makes sense to add there.

File: main/menu.js (hunk 3 above)
Comment (sndrkrly) [reply main/menu.js:217]: I also use Mac so I don't know, but I will look for the "alternative" for windows

**Comment by PalmerAL** [comment]
This was completed in #2376.

**Pull request closed without merging**